index.js

```diff
@@ -88,10 +89,10 @@ function createIsUnavailable() {
     return result;
   }
 
-  return function isUnavailable(node) {
-    if (node === document.documentElement) return false;
+  return function isUnavailable(node, elementDocument) {
```

Comment:
If `elementDocument` is already available in the closure, because it's passed to `createIsUnavailable`, it's not necessary here, it is?

Reply:
Excellent point, thank you!